src/girouette/tw/border.cljc

```diff
@@ -1,10 +1,204 @@
 (ns girouette.tw.border
-  (:require [girouette.tw.common :refer [value-unit->css]]))
+  (:require [garden.selectors :as gs]
+            [girouette.tw.common :refer [value-unit->css value->css read-number dot default-pipeline]]
+            [girouette.tw.color :refer [read-color color->css]]))
 
 (def components
-  [{:id :xxx
-    :rules "
-    xxx = <'xxx'> integer
+  [{:id     :border-radius
+    :rules  "
+    border-radius = <'rounded'>  (<'-'> border-radius-position)? (<'-'> border-radius-size)?
+    border-radius-position  = border-radius-side | border-radius-corner
+    border-radius-side = 't' | 'r' | 'b' | 'l'
+    border-radius-corner = 'tl' | 'tr' | 'br' | 'bl'
+    border-radius-size = none | full | media-query-min-width
```

Comment:
The reason `none` and `full` had their own rule was to help integrating them into value-unit couples. Here is a different context, I suggest not using the grammar rule and use a string literal directly.

Also, the media-query rule seem to match the grammar for the border-radius, but it is a coincidence. I suggest not using it.

```suggestion
    border-radius-size = 'none' | 'full' | 'sm' | 'md' | ...
```